src/manifest.ts

```diff
@@ -0,0 +1,83 @@
+import { ManifestChunk as ViteManifestChunk } from 'vite'
+
+export interface ManifestChunk extends ViteManifestChunk {
```

Comment:
I think it is safer to keep importing the type from vite.

Reply:
We could assert it in types or define util that **input manifest is of type vite**. Basically we decided to separate manifest normalization to ensure no future incompatibilities happen. If at some point, vite and this package use same manifest field differently, normalizer has to be aware of both types (while by extending, we submit to follow any additional interface changes of vite directly to the runtime)

Reply:
Sounds good. Do you feel strongly attached to `AssetMeta`? Feels like exporting Manifest/ManifestChunk seems simpler.

Reply:
Not at all about `AssetMeta` just suggesting better naming since now we can. What about `ChunkMeta` or `ResourceMeta`? `ManifestChunk` type seems it's type for the actual emited assets.

Reply:
ResourceMeta sounds good 👍